fix: Dockerize

RomanV79 opened the pull request.

zhukovsd reviewed (changes requested):
- [x] Squash 2 "fix: Dockerize" commits into 1 with a proper commit message
- [x] Add dev profile

zhukovsd committed:
feat: Dockerization

zhukovsd committed:
feat: `dev` profile

Also, `spring.thymeleaf.check-template-location` moved from `application-ide.yaml` to `application.yaml`.
